recipe/meta.yaml

```diff
@@ -28,10 +28,11 @@ requirements:
     - certifi >=2017.4.17
     - charset-normalizer >=2,<4
     - idna >=2.5,<4
-    - urllib3 >=1.21.1,<1.27
+    - urllib3 >=1.21.1,<3
```

Comment:
Because `urllib3 2.x.x` can **break** things (https://urllib3.readthedocs.io/en/latest/v2-migration-guide.html and https://urllib3.readthedocs.io/en/latest/v2-migration-guide.html) could you test downstream packages?